src/components/operations/operation-details/react/runtime/TypeDefinitions.tsx

```diff
@@ -0,0 +1,192 @@
+            <TableHeader>
+                <TableRow className={"fui-table-headerRow"}>
+                    <TableHeaderCell><Body1Strong>Type</Body1Strong></TableHeaderCell>
+                    <TableHeaderCell><Body1Strong>Values</Body1Strong></TableHeaderCell>
+                </TableRow>
+            </TableHeader>
+            <TableBody>
+                <TableRow className={"fui-table-body-row"}>
+                    <TableCell><Body1>{definition.type["name"]}</Body1></TableCell>
+                    <TableCell><Body1>{enumValues}</Body1></TableCell>
+                </TableRow>
+            </TableBody>
+        </Table>
+    );
+
+}
+
+const TypeDefinitionObject = ({ definition, showExamples, getReferenceUrl }: TypeDefinitionProps) => {
+    if (!definition.properties || definition.properties.length === 0) return;
+
+    return (
+        <Table aria-label={definition.name} className={"fui-table"}>
+            <TableHeader>
+                <TableRow className={"fui-table-headerRow"}>
+                    <TableHeaderCell><Body1Strong>Name</Body1Strong></TableHeaderCell>
+                    <TableHeaderCell><Body1Strong>Required</Body1Strong></TableHeaderCell>
+                    {definition.readOnly && <TableHeaderCell><Body1Strong>Read-only</Body1Strong></TableHeaderCell>}
+                    <TableHeaderCell><Body1Strong>Type</Body1Strong></TableHeaderCell>
+                    <TableHeaderCell><Body1Strong>Description</Body1Strong></TableHeaderCell>
+                    {showExamples && <TableHeaderCell><Body1Strong>Example</Body1Strong></TableHeaderCell>}
+                </TableRow>
+            </TableHeader>
+            <TableBody>
+                {definition.properties.map(property => {
+                    let type: JSX.Element;
+                    const typeName = property.type["name"];
+
+                    if (property.type.displayAs === TPropertyDisplayAs.combination) {
+                        let children = [<Body1 block>{property.type["combinationType"]}:</Body1>];
+
+                        property.type["combination"].map(combinationProperty => {
+                            const combinationName = combinationProperty["name"];
+
+                            if (combinationProperty["displayAs"] === TPropertyDisplayAs.reference) {
+                                children.push(
+                                    <Body1 block className={"truncate-text"}>
+                                        <Link href={getReferenceUrl(combinationName)} title={combinationName}>{combinationName}</Link>
+                                    </Body1>
+                                );
+                            } else {
+                                children.push(<Body1 block title={combinationName}>{combinationName}</Body1>);
+                            }                                
+                        });
+                        
+                        type = <>{children}</>;
```

Comment:
why not just children?

type = children;

Reply:
Children are multiple tags and I need to wrap them into a singular one.

Reply:
I think just changing TS type of variable "type" should do the same but just a nitpick, feel free to leave it as is